test/Interop/Cxx/class/nonescapable-errors.swift

```diff
@@ -147,6 +147,30 @@ using HasAnonStructNonEscapable = HasAnonStruct<NonEscapable>;
 using NonEscapableHasAnonUnionNonEscapable = NonEscapableHasAnonUnion<NonEscapable>;
 using NonEscapableOptional = std::optional<NonEscapable>;
 
+// Infered as non-escapable 
+struct Aggregate {
+  int a;
+  View b;
+  bool c;
+
+  void someMethod() {}
+}; 
+
+// This is a complex record, so we don't infer escapability.
```

Comment:
Can you leave a reminder about what makes this record complex? Is it the presence of a user-defined default constructor? Or copy constructor?